### DISCUSSION TITLE
Related to Contribution

### DISCUSSION THREAD

**Discussion by paraskamaliya** [community/community#75588] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have some collaborative projects and I have contributed enough to those projects. but the problem is in those repos I am not showing as a contributor in the contributors tab. Please help me resolve this.

**Comment by Habibsultani** [comment]
Hi, 
I just have a few suggestions to resolve this problem. 
1: Check your Git configuration and ensure that your email and username is correctly.
2: Check the repository setting and ensure you have the right access level to the project. If you don't so reach to repository owner and ask them to give the right access level.
3: Check  you pull requests and ensure that you are in the right branch.

After checking all the process still not showing you as a contributor I suggest to reach to the repository owner and maybe your team member. 
thanks
